### DISCUSSION TITLE
Relation Manager positioning/layouts?

### DISCUSSION THREAD

**Discussion by MACscr** [filamentphp/filament#3471] [Ideas]
having my groups of RM's at the bottom of the page makes it really easy for them to get lost in everything. Id love for to be able to control the overall layout of the the page and maybe do something like Tabs o Pills (that addon) so that the first tab is the regular form and the additional tabs/pills are for the RM's. Maybe even have a search option that could search that particular record and its RM's. In know thats more a secondary request, but from what I have been told, there really arent many options positioning of the RM's. Hopefully im making some sense.

**Comment by chrispian** [comment]
I love the RMs, they work really well. But users have complained about them as well being below the submit and not noticing them. Would be awesome if we could control the position of them somehow.

**Comment by robyhartonodev** [comment]
There is a possibility to put the relation manager into tab layout instead of the bottom according to the docs 

https://filamentphp.com/docs/2.x/admin/resources/relation-managers#moving-the-resource-form-to-tabs

**Comment by kayschima** [comment]
Overwriting hasCombinedRelationManagerTabsWithForm() is a nice way to save some UI space on the forms.

But this solution is not quite what I would like to implement with filament.
It would be best if the Relation Manager could also be integrated into Tabs or Wizards on the page.

**Comment by avinmaster** [comment]
Any updates here?
